src/web/mjs/connectors/MangaNel.mjs

```diff
@@ -18,10 +18,18 @@ export default class MangaNel extends Connector {
         this.queryPages = 'div.container-chapter-reader source';
     }
 
+    get tld() {
+        return new URL(this.url).hostname.split('.').slice(-2).join('.');
```

Comment:
This does not work for more complicated TLD's. Eg. with a dot. like 'co.uk'. Maybe something like [tld.js](https://github.com/oncletom/tld.js) `getDomain()` could be an option.